buildSrc/plugins/src/main/groovy/org/jetbrains/kotlin/KonanTest.groovy

```diff
@@ -593,8 +603,10 @@ class RunInteropKonanTest extends KonanTest {
 }
 
 class LinkKonanTest extends KonanTest {
```

Comment:
Wouldn't be easier to have common superclasses for all tests enabled and disabled in dev runs, i.e. `DeveloperTest` and `BuildserverTest` where this property is overriden?

Reply:
This hack was introduced long times ago and its elimination is what Pavel working on. I've added only comment, to avoid occasional removing this property, which could be caused by following Idea suggestions.